system-config/11-qubes-ignore-zvol-devices.rules

```diff
@@ -1,3 +1,4 @@
 # Determine if a zvol is a Qubes volume, then ignore it.
+KERNEL=="zd[0-9]*", SUBSYSTEM=="block", ACTION=="add|change", ENV{QUBES_IS_PART_OF_QVM_POOL}="1"
```

Comment:
This needs a comment. But also, it makes non-vm-related usage of zfs much more fragile. For example, this change on its own will break rootfs on zfs (if one manages to set it up), as `zvol_is_qubes_volume` script isn't present in initramfs. Such dependency needs to be documented somewhere.

Reply:
> this change on its own will break rootfs on zfs (if one manages to set it up)

Root on ZFS will still work, as that does not use zvols.  Root and swap on a zvol should also work (though I have not tested it) due to:

```
# Tell udev that zvols are ready.
KERNEL=="zd[0-9]*", SUBSYSTEM=="block", ACTION=="add|change", ENV{SYSTEMD_READY}="1"
```

from `99z-qubes-mark-ready.rules`.